From b3dc7f6af7f2812d5b4a42592976c5b9d7febaad Mon Sep 17 00:00:00 2001
From: wangxiao65 <wangxiao65@huawei.com>
Date: Sun, 7 Feb 2021 12:02:08 +0800
Subject: [PATCH] modify automake version
When Makefile.am is modified, automake needs to be executed.
if the version of automake in the compile enviroment is inconsistent
with the version specified in configure, the compile will fail.

---
 configure.ac | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index c879e47..8bf37b0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -48,7 +48,7 @@ LT_INIT([disable-static])
 AC_CONFIG_SRCDIR([common/flatpak-dir.c])
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE([1.13.4 no-define no-dist-gzip dist-xz tar-ustar foreign subdir-objects])
+AM_INIT_AUTOMAKE([1.16.2 no-define no-dist-gzip dist-xz tar-ustar foreign subdir-objects])
 AC_PROG_SED
 AC_PROG_YACC
 
